tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "3.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: verify-repo-signature
+      image: registry.redhat.io/rhtas/cosign-rhel9@sha256:75791f7ee9257e837d1e3db13415d858c397e803ffa4534c51ca19a2ba34b2ab
+      script: |
+        set -eux
+        MRRC_FILE="$(workspaces.data.path)/$(params.mrrcParamFilePath)"
+        . "$MRRC_FILE"
+        echo "Verify the maven repo zip $MRRC_ZIP_REGISTRY"
+        cosign verify --key /home/konflux/.cosign/cosign.pub $MRRC_ZIP_REGISTRY
+      volumeMounts:
+        - name: "pub-key-vol"
+          mountPath: "/home/konflux/.cosign/"
+    - name: download-maven-repo
+      image: quay.io/konflux-ci/oras:477ee20f7e6d146ea203c4aaa0507f4d1e925a53
```

Comment:
Could you use the release-service-utils image? It has oras in it.

Reply:
Ok will use.

Reply:
Fixed

Reply:
Cool, thanks.